cert_manager/assets/dashboards/certmanager_overview.json

```diff
@@ -0,0 +1 @@
"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"bars"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":633394604986630,"layout":{"x":0,"y":25,"width":47,"height":15},"definition":{"title":"Number of requests made by the ACME client","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"q":"sum:cert_manager.http_acme_client.request.count{*} by {host,method}.as_count()","on_right_yaxis":false,"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"bars"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":8469221575003332,"layout":{"x":-1,"y":41,"width":47,"height":15},"definition":{"title":"ACME Request Latency","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"q":"avg:cert_manager.http_acme_client.request.duration.quantile{*} by {quantile}","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":3003151349834856,"layout":{"x":0,"y":57,"width":47,"height":15},"definition":{"title":"ACME Request Errors","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"q":"sum:cert_manager.http_acme_client.request.count{status:400 OR s} by {status}.as_count()","on_right_yaxis":false,"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"bars"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":5992721770987361,"layout":{"x":49,"y":25,"width":60,"height":47},"definition":{"title_size":"16","title_align":"left","type":"log_stream","indexes":[],"query":"service:cert-manager*","sort":{"column":"time","order":"desc"},"columns":["host","service"],"show_date_column":true,"show_message_column":true,"message_display":"expanded-md"}},{"id":917472946219287,"layout":{"x":0,"y":16,"width":47,"height":8},"definition":{"type":"note","content":"ACME Client","background_color":"orange","font_size":"18","text_align":"center","vertical_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom","has_padding":true}},{"id":8624676510666531,"layout":{"x":49,"y":16,"width":60,"height":8},"definition":{"type":"note","content":"Cert Manager Logs","background_color":"orange","font_size":"18","text_align":"center","vertical_align":"center","show_tick":true,"tick_pos":"50%","tick_edge":"bottom","has_padding":true}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"id":"wzm-6tm-d6b"}
```

Comment:
The description needs to be filled out. See an [example here](https://github.com/DataDog/integrations-core/pull/6806/files#diff-e0e3367a0193faa1667f5ef4a0a446db98c2109f3029439ffe8dd43643b53340R18).

Also, having this all on 1 line makes it harder to review. The PR linked above has an example.